chainer/link.py

```diff
@@ -609,6 +609,9 @@ def serialize(self, serializer):
         d = self.__dict__
         for name in self._params:
             param = d[name]
+            param.data = (numpy.array(param.data)
```

Comment:
Could you add test case for this?

Reply:
will provide it later

Reply:
The code has been covered by your test case test_link.py

Reply:
* Let me confirm if I my understanding is correct; this intends to convert the iDeep array back to NumPy array? I'm not sure if it is needed.
* When it is needed, should it be a copy? (`numpy.asarray` cannot be used here?)

Reply:
unit test show that numpy.asarray cannot be used here.

Reply:
@kmaehashi what comment you meant? add the info "numpy.asarray cannot be used" in the code?

Reply:
@mingxiaoh Could you clarify why this is needed? (Currentl code should be fixed anyways, as it does not work when the param is on GPU).